config/routes.rb

```diff
@@ -2,7 +2,13 @@
 
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'users#index'
 
+  resources :users do
+    resources :posts do
+      resources :comments
+    end
+  end
```

Comment:
- [ ] Kindly configure your routes properly, so you don't expose unnecessary endpoints

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```